.github/workflows/main.yml

```diff
@@ -20,7 +20,9 @@ jobs:
           - python: 3.9
             image_tag: nightly
           - python: 3.6
-            image_tag: latest
+            image_tag: 3.11
```

Comment:
I know, this is a change in behavior, but I think, it is on the one hand more explicit, and on the other hand might give us a sign that we need to rebuild one of those container images for plugin bugfix releases at the right time.

cc @daviddavis